sdk/js-cloud-server/src/request.ts

```diff
@@ -0,0 +1,216 @@
+import { DVCPopulatedUser } from './models/populatedUser'
```

Comment:
any changes in here?

Reply:
No changes to the functions, just pulling out the shared methods.